MPCAutofill/MPCAutofill/settings.py

```diff
@@ -206,3 +208,20 @@
 PATREON_CLIENT = env("PATREON_CLIENT", default="")
 PATREON_SECRET = env("PATREON_SECRET", default="")
 PATREON_URL = env("PATREON_URL", default="")
+
+# Sentry
+sentry_sdk.init(

```

Comment:
yes, my sentry DSN is hard-coded into django settings. i want to track uncaught exceptions in any deployment of this project through sentry, and sentry reckons the DSN is fine to have publicly listed: https://docs.sentry.io/product/sentry-basics/dsn-explainer/